scripts/parser-tests/utils/parser-test-runner.js

```diff
@@ -1,8 +1,8 @@
 "use strict";
 
-const fs = require("fs").promises;
-const chalk = require("chalk");
-const { parse: parser } = require("../../../packages/babel-parser");
+import { promises as fs } from "fs";
+import chalk from "chalk";
+const { parse: parser } = import("../../../packages/babel-parser");
```

Comment:
I think this can be
`import { parse as parser } from ("../../../packages/babel-parser");`